app/views/hyrax/homepage/_announcement.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if display_content_block? @announcement_text %>
-  <div id="announcement" class="row">
-    <%= displayable_content_block @announcement_text, class: 'row', id: 'announcement' %>
+  <div id="announcement_text" class="row">
```

Comment:
@Dananji Do you remember why this id change is necessary or desirable?

Reply:
I'm sorry, I don't remember why I made that change.

Reply:
I found the answer when I dug back through the issue: https://github.com/samvera/hyrax/issues/3961#issuecomment-551265389

👍 I consider this id change a necessary bug fix.